fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingMutuallyByRecords.java

```diff
@@ -362,16 +371,22 @@ private CompletableFuture<Boolean> buildNextRangeOnly(List<Range> missingRanges,
                 }
                 return AsyncUtil.READY_TRUE; // let the caller handle the recovery
             }
-            Range rangeToBuild = fragmentIterationType == FragmentIterationType.FULL ?
+            final boolean isFull = fragmentIterationType == FragmentIterationType.FULL;
+            Range rangeToBuild = isFull ?
                                  fullyUnBuiltRange(missingRanges, fragmentRange) :
                                  partlyUnBuiltRange(missingRanges, fragmentRange);
-            if (LOGGER.isDebugEnabled()) {
-                LOGGER.debug(KeyValueLogMessage.of("range to build",
-                        LogMessageKeys.SCAN_TYPE, fragmentIterationType,
-                        LogMessageKeys.RANGE, rangeToBuild,
-                        LogMessageKeys.ORIGINAL_RANGE, fragmentRange));
+            if (LOGGER.isInfoEnabled()) {
+                LOGGER.info(KeyValueLogMessage.build("range to build",
```

Comment:
I'm worried this might be too chatty in general. It's probably not a big problem since building mutually is still experimental, but perhaps this indicates that we should (as a followup) add an option to change the logging level for an individual index build.

Reply:
✅ 

To reduce noice, will now log a message only when attempting to build this fragment/range.